### PR TITLE
add 110 Balanced Binary Tree

### LINKED ISSUE #47: add 110 Balanced Binary Tree
## Summary
add 110

## Goal
add 110

## Todo
- [ ] add 110
- [ ] write time and space complexity

## Deadline
4/27

## Parent issue
None

## References
None

## Notes
None

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 35m

### CODE REVIEW

**Comment by kobakobashu** [comment]
Please check my code.

File: 110.balanced-binary-tree.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        """
+        time: O(N)  N : the total of all nodes
+        space: O(N)
+        """
+        if not root:
+            return True
+        node_left = self.depth(root.left)
+        node_right = self.depth(root.right)
+        return (abs(node_left - node_right) < 2) and self.isBalanced(root.left) and self.isBalanced(root.right)
+    def depth(self, node):
```
Comment (ryuji0123) [inline 110.balanced-binary-tree.py:25]: insert a line before defining function

File: 110.balanced-binary-tree.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#         self.right = right
+class Solution:
+    def isBalanced(self, root: TreeNode) -> bool:
+        """
+        time: O(N)  N : the total of all nodes
+        space: O(N)
+        """
+        if not root:
+            return True
+        node_left = self.depth(root.left)
```
Comment (ryuji0123) [inline 110.balanced-binary-tree.py:22]: node_left is a very weird name because it is actually depth

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I fixed my code.  Please check it.

**Pull request closed without merging**